model/repo_ref.go

```diff
@@ -88,6 +89,20 @@ func FindRepoRefByOwnerAndRepo(owner, repoName string) (*RepoRef, error) {
 	}))
 }
 
+// FindRepoRefsToSync finds all repo refs that have Parameter Sore enabled but
+// don't have their project variables synced to Parameter Store yet.
+func FindRepoRefsToSync(ctx context.Context) ([]RepoRef, error) {
```

Comment:
Does DEVPROD-11882 apply to this function?

Reply:
Yes, this is temporary logic.